src/Neo.VM/Unsafe.cs

```diff
@@ -38,5 +40,40 @@ public static bool NotZero(ReadOnlySpan<byte> x)
             }
             return false;
         }
+
+        /// <summary>
+        /// Get 64-bit hash code for a byte array
+        /// </summary>
+        /// <param name="span">Span</param>
+        /// <returns></returns>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static long HashBytes(ReadOnlySpan<byte> span)
```

Comment:
Return int and avoid casts?

Reply:
it's an 64-bit hash function.

Reply:
Any other int32 hash method?

Reply:
Here a simple one. Just remember the one I in code atm is faster
```csharp
public override int GetHashCode()
{
    if (byteArray == null || byteArray.Length == 0)
        return 0;

    unchecked // Overflow is fine, just wrap
    {
        int hash = 17; // Seed with a prime number
        foreach (byte b in byteArray)
        {
            hash = hash * 31 + b; // Multiply by a prime number and add byte value
        }
        return hash;
    }
}
```